docker-compose-with-temporal.yml

```diff
@@ -130,6 +135,45 @@ services:
     privileged: true
     restart: on-failure
 
+  telemetry:
+    image: "telemetry:ra-dev-0" # TODO: change to public image
+    restart: on-failure
+    env_file: docker.env
+    ports:
+      - '9125:9125/udp'
+    command:
+      - bash
+      - -c
+      - |
+        if [[ "$${RTEL_ENABLED:-false}" == "false" ]]; then
+          echo 1>&2 "RTEL_ENABLED is not true, going to sleep."
+          sleep inf
+        else
+          exec retool-telemetry
+        fi
+    environment:
+      RTEL_DEPLOYMENT_MODE: 'docker'
```

Comment:
maybe: align this to https://github.com/tryretool/retool_development/blob/dev/backend/src/server/modules/jobsRunner/startupTelemetry.ts#L49-L62

DEPLOYMENT_TEMPLATE_TYPE
DEPLOYMENT_TEMPLATE_VERSION

in this case `docker-compose`

Reply:
ah yes, I need to address in the helm side too. will do.